### PR TITLE
bump(main/ol): 2.6

### PR DESCRIPTION
**Pull request by yuriy-chumak**
Version update.
Manually tested for x86_64 as a native dev. platform.
Manually tested for arm, aarch64, i686 with qemu-*-static.

### CODE REVIEW

File: packages/ol/no-strip.patch [hunk 1]
```diff
@@ -0,0 +1,12 @@
+diff --git a/GNUmakefile b/GNUmakefile
+index 160b044f..fcbc2cd1 100644
+--- a/GNUmakefile
++++ b/GNUmakefile
+@@ -257,7 +257,7 @@ libol.so:
+ # real name of
+ olvm: vm
+ 	cp $< $@
+-	strip $@
++	#strip $@
```
Comment (twaik) [inline packages/ol/no-strip.patch:10]: You can replace `strip` with `llvm-strip` which is already in PATH.

Comment (yuriy-chumak) [reply packages/ol/no-strip.patch:10]: Thanks. Replaced.

**Comment by Biswa96** [comment]
Next time, please remove the `TERMUX_PKG_REVISION` line after version upgrade.

**Comment by yuriy-chumak** [comment]
> Next time, please remove the `TERMUX_PKG_REVISION` line after version upgrade.

Noted. Will do.

**Pull request merged**